src/main/java/worker/Main.java

```diff
@@ -64,12 +73,14 @@ public static void startWorker(int port) {
 
     ActorSystem system = ActorSystem.create("WorkerSystem", conf);
 
-    Set<ActorSelection> initialContacts = new HashSet<ActorSelection>();
+    Set<ActorPath> initialContacts = new HashSet<>();
     for (String contactAddress : conf.getStringList("contact-points")) {
-      initialContacts.add(system.actorSelection(contactAddress + "/user/receptionist"));
+      initialContacts.add(ActorPaths.fromString(contactAddress + "/system/receptionist"));
     }
```

Comment:
The contacts can be specified in conf now, so ClusterClientSettings can load this.
https://github.com/akka/akka/blob/master/akka-cluster-tools/src/main/resources/reference.conf#L86

Since you might have several ClusterClient instances it makes most sense to have a separate config section for each:

    # default settings from akka.cluster.client section (reference.conf)
    cluster-client-1 = ${akka.cluster.client}
    cluster-client-1.initial-contacts = [
      "akka.tcp://ClusterSystem@127.0.0.1:2551/system/receptionist",
      "akka.tcp://ClusterSystem@127.0.0.1:2552/system/receptionist"]

and then use

    ClusterClientSettings.create(system.settings.config.getConfig("cluster-client-1")

instead of the programatic `.withInitialContacts`

note that this would replace the contact-points in https://github.com/typesafehub/activator-akka-distributed-workers-java/blob/master/src/main/resources/worker.conf